xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexTradeServiceRaw.java

```diff
@@ -271,17 +317,22 @@ public BitfinexOfferStatusResponse placeBitfinexFixedRateLoanOrder(
     String direction = loanOrder.getType() == OrderType.BID ? "loan" : "lend";
 
     BitfinexOfferStatusResponse newOrderResponse =
-        bitfinex.newOffer(
-            apiKey,
-            payloadCreator,
-            signatureCreator,
-            new BitfinexNewOfferRequest(
-                String.valueOf(exchange.getNonceFactory().createValue()),
-                loanOrder.getCurrency(),
-                loanOrder.getOriginalAmount(),
-                loanOrder.getRate(),
-                loanOrder.getDayPeriod(),
-                direction));
+        decorateApiCall(
+                () ->
+                    bitfinex.newOffer(
+                        apiKey,
+                        payloadCreator,
+                        signatureCreator,
+                        new BitfinexNewOfferRequest(
+                            String.valueOf(exchange.getNonceFactory().createValue()),
+                            loanOrder.getCurrency(),
+                            loanOrder.getOriginalAmount(),
+                            loanOrder.getRate(),
+                            loanOrder.getDayPeriod(),
+                            direction)))
+            .withRetry(retry("trade-loanOrder"))
```

Comment:
non-idempotent requests should not have retries on them, you don't want to mistakenly double send an order